pyrogram/client/methods/bots/callback_query/answer_callback_query.py

```diff
@@ -20,7 +20,9 @@
 from ....ext import BaseClient
 
 
-class AnswerCallbackQuery(BaseClient):
+# noinspection PyUnresolvedReferences
```

Comment:
I don't really like this either. How can we be 100% sure is pycharm's fault here?

Reply:
Official docs use it in exactly this way. Fairly sure it's a Pycharm issue, and it works flawlessly

Reply:
I lied. Official docs only say
"Changed in version 3.3: Returns the registered subclass, to allow usage as a class decorator."
-- https://docs.python.org/3/library/abc.html#abc.ABCMeta.register

Examples on the internet use it like this.